src/Codeception/Module/Yii2.php

```diff
@@ -218,7 +213,7 @@
  *       applicationClass: class-string<\yii\base\Application>|null
  *   }
  */
-class Yii2 extends Framework implements ActiveRecord, MultiSession, PartedModule
+final class Yii2 extends Framework implements ActiveRecord, PartedModule
```

Comment:
https://github.com/Codeception/module-yii2/issues/110

Reply:
This change is now in master, but not yet released. I've reopened the ticket to discuss. We can unfinalize this class if needed.